air-quality-ui/system_tests/ui/summary_page.spec.ts

```diff
@@ -1,78 +1,570 @@
+            createMeasurementSummaryAPIResponseData({
+              measurement_base_time: '2024-07-08T03:00:00Z',
+            }),
+            createMeasurementSummaryAPIResponseData({
+              measurement_base_time: '2024-07-08T12:00:00Z',
+              overall_aqi_level: { mean: 3 },
+              no2: { mean: { aqi_level: 3, value: 119 } },
+              o3: { mean: { aqi_level: 3, value: 129 } },
+              pm2_5: { mean: { aqi_level: 3, value: 24 } },
+              pm10: { mean: { aqi_level: 3, value: 49 } },
+              so2: { mean: { aqi_level: 3, value: 349 } },
+            }),
+          ]
+
+          await summaryPage.setupPageWithMockData(
+            forecastLondonValidTimeArray,
+            measurementsLondonArray,
+          )
+
+          const expectedTableContents: string[][] = [
+            [
+              '3',
+              '3',
+              '0',
+              '22.5',
+              '24',
+              '08 Jul 12:00',
+              '45.5',
+              '49',
+              '08 Jul 12:00',
+              '100.5',
+              '119',
+              '08 Jul 12:00',
+              '100.5',
+              '129',
+              '08 Jul 12:00',
+              '300.5',
+              '349',
+              '08 Jul 12:00',
+            ],
+          ]
+
+          await summaryPage.assertGridValues(expectedTableContents)
+        })
+
+        test('Verify the forecast AQI Level value is the highest overall AQI level in forecast response', async ({
+          summaryPage,
+        }) => {
+          const forecastLondonValidTimeArray: object[] = [
+            createForecastAPIResponseData(),
+            createForecastAPIResponseData({
+              valid_time: '2024-07-08T03:00:00Z',
+            }),
+            createForecastAPIResponseData({
+              valid_time: '2024-07-08T12:00:00Z',
+            }),
+          ]
+
+          const measurementsLondonArray: object[] = [
+            createMeasurementSummaryAPIResponseData(),
+            createMeasurementSummaryAPIResponseData({
+              measurement_base_time: '2024-07-08T03:00:00Z',
+            }),
+            createMeasurementSummaryAPIResponseData({
+              measurement_base_time: '2024-07-08T12:00:00Z',
+              overall_aqi_level: { mean: 3 },
+              no2: { mean: { aqi_level: 3, value: 119 } },
+              o3: { mean: { aqi_level: 3, value: 129 } },
+              pm2_5: { mean: { aqi_level: 6, value: 799 } },
+              pm10: { mean: { aqi_level: 3, value: 49 } },
+              so2: { mean: { aqi_level: 3, value: 349 } },
+            }),
+          ]
+
+          await summaryPage.setupPageWithMockData(
+            forecastLondonValidTimeArray,
+            measurementsLondonArray,
+          )
+
+          const expectedTableContents: string[][] = [
+            [
+              '3',
+              '6',
+              '-3',
+              '22.5',
```

Comment:
I don't think we really care about the data after here in this test? Can we just add the three values and leave the rest of the array blank?